salt/utils/timed_subprocess.py

```diff
@@ -86,7 +86,7 @@ def run(self):
         '''
         def receive():
             if self.with_communicate:
-                self.stdout, self.stderr = self.process.communicate(input=self.stdin)
+                self.stdout, self.stderr = self.process.communicate(input=self.stdin, timeout=self.timeout)
```

Comment:
This is causing a lint error. On further investigation, PY2 does not have the `timeout` parameter.